fendermint/ipc/src/pof.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+//! IPC proof of finality related functions
+
+use crate::Config;
+use crate::{BlockHeight, ParentViewProvider};
+use anyhow::anyhow;
+use ipc_sdk::cross::CrossMsg;
+use ipc_sdk::ValidatorSet;
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+
+/// The proof for POF.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct IPCParentFinality {
+    /// The latest chain height
+    pub height: u64,
+    /// The block hash. For FVM, it is a Cid. For Evm, it is bytes32.
```

Comment:
I'm not sure why we mention EVM here. Fendermint will have to deal with Tendermint parents, but not `geth`.

Reply:
It is possible to have a purely EVM subnet, just be general here.